src/components/HeaderComponent.js

```diff
@@ -0,0 +1,18 @@
+import { Header } from 'react-bootstrap';
+import React, { Component } from 'react';
+
+import '../assets/App.css';
+import '../assets/index.css';
+
+class CustomHeader extends Component {
```

Comment:
For components that only have a render method without any business logic, you should consider using _functional components_ instead

Reply:
Okay. And I will import them the same way?